lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -42,11 +42,11 @@ def engine_running(engine, &block)
           end
         end
 
-        def failed(output)
-          spinner.stop("Failed")
-          puts colorize("\nAnalysis failed with the following output:", :red)
+        def errored(output)
+          spinner.stop("Errored")
+          puts colorize("\nAnalysis errored with the following output:", :red)
           puts output
-          exit 1
+          exit 2
```

Comment:
I guess what I'm saying is, aside from analyze and validate config, there really is no other cases where the CLI would be involved:

- `G*` these are a clone in a totally different container
- `E*` these are just error vs timeout which builder knows (error is non-zero exit, timeout is no exit at at all, same for output exceeded, etc)

Not super important now; 170 for issues-found is fine with me regardless of future planning.